awscli/botocore/auth.py

```diff
@@ -436,11 +435,13 @@ def add_auth(self, request):
         self._inject_signature_to_request(request, signature)
 
     def _inject_signature_to_request(self, request, signature):
-        l = [f'AWS4-HMAC-SHA256 Credential={self.scope(request)}']
+        auth_str = [f'AWS4-HMAC-SHA256 Credential={self.scope(request)}']
```

Comment:
This was an example of https://docs.astral.sh/ruff/rules/ambiguous-variable-name/